src/opower/utilities/evergy.py

```diff
@@ -23,19 +23,14 @@ def handle_starttag(self, tag: str, attrs: list[tuple[str, str | None]]) -> None
             self.verification_token = token
 
 
-class Evergy(UtilityBase):
+class __EvergyBase(UtilityBase):
     """Evergy."""
 
     @staticmethod
     def name() -> str:
         """Distinct recognizable name of the utility."""
         return "Evergy"
```

Comment:
The home assistant integration is using this name as key. So each subclass needs to have a distinct name. Instead of creating two subclasses, isn't there any response that indicates which domain the browser should connect to?

Reply:
Took a closer look and yep, looks like there's a request to `https://www.evergy.com/sc-api/account/getaccountpremiseselector`.